openshift/system/Dockerfile.on_prem

```diff
@@ -31,6 +31,7 @@ ADD lib/developer_portal/*.gemspec lib/developer_portal/
 ADD vendor/active-docs/*.gemspec vendor/active-docs/
 ADD Gemfile* ./
 ADD Rakefile ./
+ADD .ruby-version ./
```

Comment:
@guicassolato shall i remove this again now that it seems the problem was with ` .dockerignore` file?

Reply:
Yes, the file should be there despite of that.